packages/core/database/src/dialects/dialect.ts

```diff
@@ -17,7 +17,8 @@ export default class Dialect {
     this.client = client;
   }
 
-  configure() {}
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
time to use an abstract class & methods ?

Reply:
Almost time 😆 I would just wait until we do some investigation on supporting dialects via user code